tests/test_fuzz.py

```diff
@@ -40,7 +40,7 @@
 	else:
 		from context import multicast  # pylint: disable=cyclic-import - skipcq: PYL-R0401
 		from context import unittest
-		from hypothesis import given, strategies as st
+		from hypothesis import given, reproduce_failure, settings, strategies as st
```

Comment:
Oops! 🙉 This caused a [style regression](https://github.com/reactive-firewall/multicast/actions/runs/11136084106/attempts/1#summary-30947291181) ∵ `reproduce_failure` is unused.